scripts/load_aks_credentials.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+# Function to check if a command exists
+not_installed() {
+    ! type "$1" &> /dev/null
+}
+
+# Check for jq
+if not_installed jq; then
+    echo "jq is not installed. You can install it from: https://stedolan.github.io/jq/download/"
+    exit 1
+fi
+
+# Check for yq
+if not_installed yq; then
+    echo "yq is not installed. You can install it from: https://github.com/mikefarah/yq"
+    exit 1
+fi
+
+# Check for op
+if not_installed op; then
+    echo "op is not installed. You can install it from: https://developer.1password.com/docs/cli/get-started/"
+    exit 1
+fi
+
+# The string to match with context.user
+USERNAME="$1"
+
+# Ensure script is called with one argument
+if [ -z "$1" ]; then
+    echo "No username provided. Please provide it."
+    echo "eg. ./load_aks_credentials.sh <username>"
+    exit 1
+fi
+
+# Fetch data from 1Password
+jsonData=$(op item get $USERNAME  --fields label=CLIENT_CERTIFICATE_DATA,label=USER_NAME,label=CLIENT_KEY_DATA,label=TOKEN,label=SERVER,label=CLUSTER_NAME,label=CERTIFICATE_AUTHORITY_DATA --format json)
+
+# Check exit status of the last command
+if [ $? -ne 0 ]; then
+    exit 1
+fi
+
+# Assign the JSON values to variables
+CLIENT_CERTIFICATE_DATA=$(echo $jsonData | jq -r '.[] | select(.label=="CLIENT_CERTIFICATE_DATA") | .value')
+CLIENT_KEY_DATA=$(echo $jsonData | jq -r '.[] | select(.label=="CLIENT_KEY_DATA") | .value')
+TOKEN=$(echo $jsonData | jq -r '.[] | select(.label=="TOKEN") | .value')
+SERVER=$(echo $jsonData | jq -r '.[] | select(.label=="SERVER") | .value')
+CLUSTER_NAME=$(echo $jsonData | jq -r '.[] | select(.label=="CLUSTER_NAME") | .value')
+CERTIFICATE_AUTHORITY_DATA=$(echo $jsonData | jq -r '.[] | select(.label=="CERTIFICATE_AUTHORITY_DATA") | .value')
+USER_NAME=$(echo $jsonData | jq -r '.[] | select(.label=="USER_NAME") | .value')
+
+# Check if the file does not exist
+if [ ! -f ~/.kube/config ]; then
+    # Create a new kubeconfig file
+    CONFIG_TEMPLATE="apiVersion: v1
```

Comment:
MAYBE: to simply we could directly use
```sh
kubectl config set-cluster "$CLUSTER_NAME" --server="$SERVER" --certificate-authority-data="$CERTIFICATE_AUTHORITY_DATA"
kubectl config set-credentials "$USER_NAME" --client-certificate-data="$CLIENT_CERTIFICATE_DATA" --client-key-data="$CLIENT_KEY_DATA" --token="$TOKEN"
kubectl config set-context "$CLUSTER_NAME" --cluster="$CLUSTER_NAME" --user="$USER_NAME" --namespace=default
kubectl config use-context "$CLUSTER_NAME"
```

which we could take if there 

1. are duplicate entries 
2. and would also automatically create kubeconfig file (need to verify file creation)

Thoughts?

Reply:
Done by [ed62768](https://github.com/OpenMined/PySyft/pull/8199/commits/ed6276899c26e419269f1c1bc5d0253d916bcebc)